solr/config/schema.xml

```diff
@@ -59,32 +59,32 @@
 
   <types>
     <fieldType name="string" class="solr.StrField" sortMissingLast="true" />
-    <fieldType name="boolean" class="solr.BoolField" sortMissingLast="true"/>    
+    <fieldType name="boolean" class="solr.BoolField" sortMissingLast="true"/>
```

Comment:
I assume 99% of these diffs are whitespace errors, minus the `<solrQueryParser defaultOperator="OR"/>`?